octobot_trading/personal_data/positions/position.py

```diff
@@ -41,6 +41,7 @@ def __init__(self, trader, symbol_contract):
         self.logger_name = None
         self.position_id = None
         self.timestamp = 0
+        self.first_entry_time = 0
```

Comment:
Why not using `creation_time` and set `creation_time` at first entry instead of object creation time? Because `creation_time` is useless for now